extensions/positron-assistant/src/participants.ts

```diff
@@ -213,9 +213,12 @@ abstract class PositronAssistantParticipant implements IPositronAssistantPartici
 				const hasSelection = inEditor && request.location2.selection?.isEmpty === false;
 				const isEditMode = this.id === ParticipantID.Edit;
 				const isAgentMode = this.id === ParticipantID.Agent;
+				const isNotebookSession = positronContext.activeSession?.mode === positron.LanguageRuntimeSessionMode.Notebook;
+				const isStreamingInlineEditor = isStreamingEditsEnabled() && (this.id === ParticipantID.Editor || this.id === ParticipantID.Notebook);
```

Comment:
Include Notebook participants as well, to match the regular Editor inline chat experience (no tools!)